tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
```

Comment:
The answer is code is copyright the institution that employed the person writing it. AURA/LSST isn't even a legal entity that can own copyright. The confusion is over the RFC limbo.